docs/01_setup.md

```diff
@@ -130,6 +96,71 @@ pnpm deploy-poll:NETWORK
 
 See [MACI docs](https://maci.pse.dev/docs/v1.2/integrating#deployment) for more information.
 
+## 3. Configuration
+
+The `.env.example` file contains instructions for most of these steps.
+
+At the very minimum you need to configure a subgraph, nextauth, admin address, MACI address, the EAS Schema and the voting periods under App Configuration.
+
+#### Subgraph
+
+In the MACI repo and head to the subgraph folder.
+
+> [!IMPORTANT]
+> The following steps focus on The Graph, but you can deploy a subgraph using Alchemy.
+
+1. Make sure you have `{network}.json` file in `config` folder, where network is a CLI name supported for subgraph network [https://thegraph.com/docs/en/developing/supported-networks/](https://thegraph.com/docs/en/developing/supported-networks/).
+
+2. Create subgraph in [the graph studio](https://thegraph.com/studio/).
+
+3. Add network, maci contract address and maci contract deployed block.
+
+```json
+{
+  "network": "optimism-sepolia",
+  "maciContractAddress": "0xD18Ca45b6cC1f409380731C40551BD66932046c3",
+  "maciContractStartBlock": 11052407
+}
+```
+
+4. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
+    - Create an `.env` file, and run the follow command in the console `export $(xargs <.env)`
+5. Run `graph auth --studio {key}`. You can find the key in subgraph studio dashboard.
+6. Run `pnpm run deploy` to deploy subgraph
+
+#### Auth
+
+1. Generate a secret (`openssl rand -base64 32`)
+2. Configure `NEXTAUTH_URL` (this should only be necessary on localhost or for production domains)
+
+#### Network
+
+The default configuration is Optimism Sepolia for development and Optimism for production.
+
+You can find supported networks on the EAS documentation website: https://docs.attest.sh/docs/quick--start/contracts
+
+#### App
+
+Configure how many votes each voter receives and how many votes each project can receive.
+You can also find configurations for when voting starts and ends as well as the registration and review period.
```

Comment:
This sentence is weird for me. So the voting actually starts after `NEXT_PUBLIC_REGISTRATION_END_DATE` and as soon as it found there is a poll in the maci contract. The end date of the voting depends on the contract.